apsuite/commisslib/meas_touschek_lifetime.py

```diff
@@ -70,23 +78,43 @@ def __str__(self):
 
 
 class MeasTouschekLifetime(_BaseClass):
-    """."""
+    """Measurement of two single-bunches current decay with BPM sum signal.
+
+    In single-bunch mode, RFFE attenuation must be adjusted in all BPMs
+    before injection, the default value is 30dB. The BPM used to be measured
+    is allowed to have an attenuation of 14dB to increase the sum signal. In
+    the case that some BPMs attenuation setting fails, a local reset of RFFE
+    module at the BPM rack must be done.
+
+    The measurement is typically performed with two single-bunches with
+    different currents: one with high current (~2mA) and one with low current
+    (~0.2mA).
+
+    If correct_orbit=True in params, SOFB must be properly configured:
```

Comment:
`properly configured by hand prior to the start of the experiment.`

Reply:
ok